### PR TITLE
Feature/gps bug

### PR DESCRIPTION
**Pull request by tejassonar**
Fixes #

Changes in this pull request:
- GPS issue of timeout solved for Ios (on 'Ask Next Time' app should ask permission again and going back from the 'Settings' it should go to the 'Tree Inventory'. Also, the mapMarking screen freezing issue [could not reproduce it])
- alertModal padding
- decreased animation duration for the marker to 1 second

@

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -678,4 +678,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 801f9c43586733b267138576ff038f2dee20112b
 
-COCOAPODS: 1.10.1
+COCOAPODS: 1.10.0
```
Comment (norbertschuler) [inline ios/Podfile.lock:681]: Can you please also update your [CocoaPods installation](https://guides.cocoapods.org/using/getting-started.html) to version 1.10.1.

**Comment by norbertschuler** [comment]
> * GPS issue of timeout solved for Ios

I have no idea what was wrong before. If you could explain it, I would help you testing and approve this PR :-)

File: app/components/RegisterSingleTree/MapMarking.js [hunk 2]
```diff
@@ -372,7 +379,8 @@ const MapMarking = ({ updateScreenState, inventoryState, resetRouteStack }) => {
           if (IS_ANDROID) {
             updateCurrentPosition();
           } else {
-            Linking.openURL('app-settings:');
+            Linking.openURL('App-Prefs:LOCATION_SERVICES');
```
Comment (norbertschuler) [inline app/components/RegisterSingleTree/MapMarking.js:382]: I have not yet tested on Android, but on an iPhone 11 emulator this leads a native crash of the emulator:

Bildschirmfoto 2021-03-08 um 15 34 02

Comment (norbertschuler) [reply app/components/RegisterSingleTree/MapMarking.js:382]: Also an interesting source about this issue: https://developer.apple.com/forums/thread/108866 
Quote: _"All such prefs: and similar URL schemes are private API. Your app will be rejected, even if you did find something that happened to work this week."_ :-)

Comment (norbertschuler) [reply app/components/RegisterSingleTree/MapMarking.js:382]: Please let's just revert it to `app-settings:`.

Comment (tejassonar) [reply app/components/RegisterSingleTree/MapMarking.js:382]: sure, if that works.

**Comment by norbertschuler** [comment]
- [ ] While testing I just managed to create this crash (camera permission on my emulator was granted):

Simulator Screen Shot - iPhone 11 - 2021-03-08 at 15 45 32

**Review by norbertschuler — APPROVED** [review]

**Comment by tejassonar** [comment]
> * [ ]  While testing I just managed to create this crash (camera permission on my emulator was granted):
> 
> Simulator Screen Shot - iPhone 11 - 2021-03-08 at 15 45 32

Can you tell the steps to reproduce this error?

**Comment by norbertschuler** [comment]
> > * [ ]   While testing I just managed to create this crash (camera permission on my emulator was granted):
> Can you tell the steps to reproduce this error?

I can't. Sometimes when navigating back it happened. I just added a safety check now - better be safe than sorry.

**Comment by tejassonar** [comment]
> > > * [ ]    While testing I just managed to create this crash (camera permission on my emulator was granted):
> > >   Can you tell the steps to reproduce this error?
> 
> I can't. Sometimes when navigating back it happened. I just added a safety check now - better be safe than sorry.

Yess. Better

**Pull request merged**